news/2022-07-21-osu-philippines-conquest-ph-2022.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "osu! Philippines: CONQuest PH 2022"
+date: 2022-07-21 00:00:00 +0000
+---
+
+A monumental first for osu! in the Philippines, an osu!booth will be open at [CONQuest PH 2022](https://conquestph.com), held at the SMX Convention Centre, Pasay City, Philippines.
+
+The first ever LAN event, the **Quickfire Showdown**, will be hosted at the booth! 
+
+![Quickfire Showdown Banner](/wiki/shared/news/2022-07-21-osu-philippines-conquest-ph-2022/quickfire_showdown_banner.png) 
+
+As an extra special addition, [BTMC](https://osu.ppy.sh/users/3171691) will be present at the event playing with the talented players of the osu!Philippines community! 
+
+## Tournament
+
+The Quickfire Showdown, the main event of CONQuest PH 2022, is an open tournament held on both days for those present at the venue. As the name implies, there will be no warmups, and the mappools will be revealed on the spot. 
+
+Additionally, as a CONQuest PH 2022 exclusive, there will be a track from one of osu!'s featured artists, [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88)! 
+
+## Prizes
+
+### July 23, 2022 (Day 1)
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge + osu!merch + 3 months of osu!supporter |
+| Runner-up | osu!merch |
```

Comment:
might wanna use the silver crown for runner-up?

Reply:
looks good, sure!